From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Fri, 9 Dec 2016 15:40:58 -0500
Subject: [PATCH] Add UNUSED define

Extracted from: "Add blscfg command support to parse BootLoaderSpec config
 fragments"
---
 include/grub/compiler.h      |   2 +
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/include/grub/compiler.h b/include/grub/compiler.h
index c9e1d7a73dc..9859ff4cc79 100644
--- a/include/grub/compiler.h
+++ b/include/grub/compiler.h
@@ -48,4 +48,6 @@
 #  define CLANG_PREREQ(maj,min) 0
 #endif
 
+#define UNUSED __attribute__((__unused__))
+
 #endif /* ! GRUB_COMPILER_HEADER */
